sympy/stats/rv_interface.py

```diff
@@ -396,6 +396,63 @@ def median(X, evaluate=True, **kwargs):
         return result
     raise NotImplementedError("The median of %s is not implemeted."%str(pspace(X)))
 
+
+def coskewness(X, Y, Z, condition=None, **kwargs):
+    r"""
+    Calculates the co-skewness of three random variables.
+    Mathematically Coskewness is defined as
+
+    .. math::
+        coskewness(X,Y,Z)=\frac{E[(X-E[X]) * (Y-E[Y]) * (Z-E[Z])]} {\sigma_{X}\sigma_{Y}\sigma_{Z}}
+
+    Parameters
+    ==========
+
+    X : RandomSymbol
+            Random Variable used to calculate coskewness
+    Y : RandomSymbol
+            Random Variable used to calculate coskewness
+    Z : RandomSymbol
+            Random Variable used to calculate coskewness
+    condition : Expr containing RandomSymbols
```

Comment:
I can’t see any example of usage for condition. Can you find an example usage about this

Reply:
Sure!